pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/MultiStageEngineIntegrationTest.java

```diff
@@ -971,7 +971,7 @@ Object[][] polymorphicScalarComparisonFunctionsDataProvider() {
     inputs.add(new Object[]{"INT", "1", "0", "1"});
     inputs.add(new Object[]{"LONG", "12345678999", "12345678998", "12345678999"});
     inputs.add(new Object[]{"FLOAT", "CAST(1.234 AS FLOAT)", "CAST(1.23 AS FLOAT)", "1.234"});
-    inputs.add(new Object[]{"DOUBLE", "1.234", "1.23", "1.234"});
+    inputs.add(new Object[]{"DOUBLE", "CAST(1.234 AS DOUBLE)", "CAST(1.23 AS DOUBLE)", "1.234"});
```

Comment:
Just curious, do we still need this change?

Reply:
Ah, good catch, we don't.